modules/pruning-images.adoc

```diff
@@ -28,20 +28,39 @@ By default, the integrated registry caches blobs metadata to reduce the number
 of requests to storage, and increase the speed of processing the request.
 Pruning does not update the integrated registry cache. Images pushed after
 pruning that contain pruned layers will be broken, because the pruned layers
-that have metadata in the cache will not be pushed. Therefore, it is necessary
-to clear the cache after pruning. This can be accomplished by redeploying the
-registry:
+that have metadata in the cache will not be pushed. To avoid this issue,
+you can set the image registry to `readOnly: true` before pruning and 
+reset to `false` when complete.
 
 ----
-$ oc rollout latest dc/docker-registry
+$ oc edit configs.imageregistry.operator.openshift.io/cluster
```

Comment:
nit: `oc patch ...` so you don't have to list the unrelated stuff like proxy settings?